index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<link rel="stylesheet" href="style.css">
+<head><a id="voltar" href="index.html">Voltar</a>
+<a href="sobrenos.html">Sobre nós</a>
+<script src="./scripts.js"></script>
+
+</head>
+<body>
+    <img src="./atividadebackenddownload.jpg">
+    <h4> </h4>
```

Comment:
Sugestão: trocar o h4 por p.
```suggestion
    <p> </p>
```